code/modules/modular_computers/file_system/programs/security/camera.dm

```diff
@@ -37,7 +37,7 @@
 	size = 12
 	available_on_ntnet = TRUE
 	requires_ntnet = TRUE
-	required_access_download = access_heads
+	// required_access_download = access_heads
```

Comment:
Don't comment this, delete the line instead.

Reply:
Oh! I meant to delete this after the fact; I'd commented this while doing tests and forgot about it. I'll get to this in a bit.